Sources/DateScrubber.swift

```diff
@@ -91,9 +131,24 @@ public class DateScrubber: UIViewController {
         self.setFrame(atYpos: yPos)
     }
 
-    public func updateSectionTitle(title : String){
-        self.sectionLabel.setText(title)
-        self.setSectionlabelFrame()
+    public func updateSectionTitle(title: String) {
+
+        if self.currentSectionTitle != title {
+            self.currentSectionTitle = title
+
+            self.sectionLabel.setText(title)
+            self.setSectionlabelFrame()
+        }
+    }
+
+    private func userInteractionOnScrollViewDetected(){
+
+        NSObject.cancelPreviousPerformRequestsWithTarget(self, selector: #selector(hideScrubber), object: nil)
```

Comment:
Aha! I guess I'll try and use an operation queue then, as mentioned in the PR description I have a weird bug doing this. maybe the queue will solve that 💅